### ISSUE TITLE
Remove Cron as the focus for schedules

### ISSUE THREAD

**Issue by pedrotski** [wisp-gg/issue-tracker#17]
### Describe the feature and why you would like to see it.

Cron means nothing to a beginner server owner. A simple way around this is to create a Cron generator that is the primary focus, but allow the hardcore users to toggle into the original Cron mode. 

Here are some quick examples.

![Easy-Cron-Scheduler-1](https://github.com/user-attachments/assets/9679c8c0-b636-4c07-869f-fd1cd5f69f11)

![Easy-Cron-Schedule](https://github.com/user-attachments/assets/9dd7b44c-98f4-453d-bcbf-03bb1c5878cb)

Also if the time they select is their local time and then converted into the server time. At the moment you need to input the server time, and then it gets converted to their local time afterward.

### Describe the solution you'd like.

as above

**Comment by Lunaversitay** [comment]
I agree, the current system is too advanced for a majority of server owners to understand, a simplified version with the same functionality would be more desirable 👍
